unity/Assets/Scripts/AgentManager.cs

```diff
@@ -419,7 +419,9 @@ private enum serverTypes { WSGI, FIFO };
         float fieldOfView,
         string skyboxColor,
         bool? orthographic,
-        float? orthographicSize
+        float? orthographicSize,
+        float? nearClippingPlane = null,
+        float? farClippingPlane = null
```

Comment:
Can
```c#
        float? nearClippingPlane = null,
        float? farClippingPlane = null
```
not have defaults for the private method?

Perhaps this is just a personal style preference, but since this is a private method, I generally like how every single call to it must specify all parameters.

Here, if new parameters are added, like `nearClippingPlane` and `farClippingPlane`, they have to explicitly be passed in at every place the private method is called, and if they're not, a compiler warning will appear.

In short, I think this helps ensure that public parameters are always passed to the private method.

Reply:
sounds good, changes in